tests/Update-DbaBuildReference.Tests.ps1

```diff
@@ -0,0 +1,27 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
+    Context "Validate parameters" {
+        It "Should only contain our specific parameters" {
+            [array]$params = ([Management.Automation.CommandMetaData]$ExecutionContext.SessionState.InvokeCommand.GetCommand($CommandName, 'Function')).Parameters.Keys
+            [object[]]$knownParameters = 'EnableException'
+            Compare-Object -ReferenceObject $knownParameters -DifferenceObject $params | Should -BeNullOrEmpty
+        }
+    }
+}
+
+Describe "$CommandName Unit Test" -Tags Unittest {
+    Context "not much" {
+        It "calls the internal function" {
+            function Get-DbaBuildReferenceIndexOnline { }
+            Mock Get-DbaBuildReferenceIndexOnline -MockWith { } -ModuleName dbatools
+            { Update-DbaBuildReference } | Should -Not -Throw
```

Comment:
this one should also have `-EnableException -ErrorAction Stop` to be able to throw when an error is occurred

Reply:
you're absolutely right, thanks for the catch